### PR TITLE
[Doc] update install doc

### PR DESCRIPTION
**Pull request by liuhuiCNN**
update install doc

### CODE REVIEW

File: docs/tutorials/QUICK_STARTED.md [hunk 1]
```diff
@@ -1,70 +1,82 @@
...
 
-## Data Preparation
-
-Dataset refers to [Kaggle](https://www.kaggle.com/mbkinaci/fruit-images-for-object-detection), which contains 240 images in train dataset and 60 images in test dataset. Data categories are apple, orange and banana. Download [here](https://dataset.bj.bcebos.com/PaddleDetection_demo/fruit-detection.tar) and uncompress the dataset after download, script for data preparation is located at [download_fruit.py](../../dataset/fruit/download_fruit.py). Command is as follows:
-
-```bash
-python dataset/fruit/download_fruit.py
+## 一、Quick Start
+```
+# predict an image using pp-yolo
```
Comment (qingqing01) [inline docs/tutorials/QUICK_STARTED.md]: pp-yolo -> PP-YOLO

File: docs/tutorials/QUICK_STARTED.md [hunk 2]
```diff
@@ -1,70 +1,82 @@
...
+#
+python dataset/roadsign_voc/download_roadsign_voc.py
+```
 
-```bash
-python -u tools/train.py -c configs/yolov3_mobilenet_v1_fruit.yml \
-                        --use_vdl=True \
-                        --vdl_log_dir=vdl_fruit_dir/scalar \
-                        --eval
+## 三、Train、Eval、Infer
```
Comment (qingqing01) [inline docs/tutorials/QUICK_STARTED.md]: Remove 三

File: docs/tutorials/QUICK_STARTED.md [hunk 3]
```diff
@@ -1,70 +1,82 @@
...
 export CUDA_VISIBLE_DEVICES=0
 ```
 
-## Data Preparation
-
-Dataset refers to [Kaggle](https://www.kaggle.com/mbkinaci/fruit-images-for-object-detection), which contains 240 images in train dataset and 60 images in test dataset. Data categories are apple, orange and banana. Download [here](https://dataset.bj.bcebos.com/PaddleDetection_demo/fruit-detection.tar) and uncompress the dataset after download, script for data preparation is located at [download_fruit.py](../../dataset/fruit/download_fruit.py). Command is as follows:
-
-```bash
-python dataset/fruit/download_fruit.py
+## 一、Quick Start
```
Comment (qingqing01) [inline docs/tutorials/QUICK_STARTED.md]: Remove  一

File: docs/tutorials/QUICK_STARTED.md (hunk 2 above)
Comment (liuhuiCNN) [reply docs/tutorials/QUICK_STARTED.md]: done.

File: docs/tutorials/QUICK_STARTED.md (hunk 3 above)
Comment (liuhuiCNN) [reply docs/tutorials/QUICK_STARTED.md]: done

File: docs/tutorials/QUICK_STARTED.md (hunk 1 above)
Comment (liuhuiCNN) [reply docs/tutorials/QUICK_STARTED.md]: done

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**